src/setupTest.js

```diff
@@ -0,0 +1,9 @@
+// TODO: @jaebradley refactor this when Jest issue (https://github.com/facebook/jest/issues/4545#issuecomment-333004504) is patched
```

Comment:
Ooh, this is gnarly but looks like it's the agreed-upon workaround. Will keep an eye on that issue.